docs/developer-docs/latest/setup-deployment-guides/configurations.md

```diff
@@ -1575,3 +1575,124 @@ module.exports = () => ({
   },
 });
 ```
+
+### Role-Based Access Control <BronzeBadge link="https://strapi.io/pricing"/> <SilverBadge link="https://strapi.io/pricing"/> <GoldBadge link="https://strapi.io/pricing" withLinkIcon/>
+
+::: warning 🚧 This API is considered unstable for now.
+<br>
+:::
+
+Role-Based Access Control (RBAC) is an approach to restricting access to some users. In a Strapi application, users of the admin panel are administrators. Their roles and permissions are [configured in the admin panel](/user-docs/latest/users-roles-permissions/configuring-administrator-roles.md). The Community Edition of Strapi offers 3 default roles (Author, Editor, and Super Admin). To go further, creating custom conditions for permissions applicable to any user is also possible. This requires an Enterprise Edition with at minimum a Bronze licence plan.
+
+#### Declaring new conditions
+
+Declare a single condition as an object, and multiple conditions as an array of objects. Each condition object can have 5 possible properties:
+
+- `displayName` (string): the condition name as shown in the admin panel,
+- `name` (string): the condition name, kebab-cased,
+- `category` (string, _optional_): conditions can be grouped into categories available [in the admin panel](/user-docs/latest/users-roles-permissions/configuring-administrator-roles.md#setting-custom-conditions-for-permissions); if undefined, the condition will appear under the "Default" category,
+- `plugin` (string, _optional_): if the condition is created by a plugin, should be the plugin's name, kebab-cased (e.g `content-manager`),
+- `handler`: a query object or a function used to verify the condition (see [using the condition handler](#using-the-condition-handler))
+
+Declare and register conditions in your [`./config/functions/bootstrap.js`](/developer-docs/latest/setup-deployment-guides/configurations.md#bootstrap) file (see [Registering conditions](#registering-conditions)).
+
+:::tip NOTE
+The condition `name` property acts as a [unique id](https://github.com/strapi/strapi/blob/master/packages/strapi-admin/services/permission/condition-provider.js#L22) within its namespace, that is either the plugin if the `plugin` property is defined, or the root namespace.
+:::
+
+#### Using the condition handler
+
+The condition `handler` is used to verify the condition on the entities you read, create, update or delete. It can be a query object or a function.
```

Comment:
In fact, a condition can be applied to every permission.
The **condition query object**, however, will only work for permission attached to a particular subject (aka entity).

For instance, I could create a condition that allows me to access a particular page of the admin panel only if the server date is 5pm. In this scenario, my action is not related to any subject/entity and my condition would look like this: `handler: () => new Date().getHours() === 17`

Reply:
Oh, I seeeeee! Thank you very much for clarifying, JS! Really interesting example :-)

So I updated the content with [this commit](https://github.com/strapi/documentation/commit/61cfac32647434e857a8c91cda05edda0a7dbde8)